pyproject.toml

```diff
@@ -50,12 +50,10 @@ keywords = ["evaluation", "nlp", "llm"]
 dependencies = [
     # Base dependencies
     "transformers>=4.38.0",
-    "huggingface_hub==0.20.3",
+    "huggingface_hub>=0.20.3",
```

Comment:
I think it's generally good to not pin `hfh` (like we do in our other HF libs). Incidentally, it's a core dep of `transformers` and could arguably be removed here altogether

Reply:
we wanted to constrain a minimal version for some features :)